### ISSUE TITLE
#progressContainer needs tabindex

### ISSUE THREAD

**Issue by bearfriend** [Polymer/paper-progress#14]
I did some searching but didn't find anything on this, so excuse me if the issue is my ignorance, but it would seem that `div#progressContainer` should have `tabindex=0` so that a screenreader can focus it and get a report of the progress.

EDIT: Nevermind! I just need to get better at using screenreaders. I was using Chrome, but VoiceOver in Safari has no issues.
